app/jobs/regular/remove_banner.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  class RemoveBanner < ::Jobs::Base
+
+    def execute(args)
+      topic_id = args[:topic_id]
+
+      raise Discourse::InvalidParameters.new(:topic_id) unless topic_id.present?
```

Comment:
Is there any point in raising an error here? It will just make errors in the logs we can do nothing about. Better to just do an early return.

Reply:
same behavior than remove pin job